.github/workflows/pull-request.yml

```diff
@@ -113,3 +103,18 @@ jobs:
         with:
           path: './coverage/lcov.info'
           min_coverage: 94.3
+
+  JSDocs:
+    name: 'JSDocs comments and pipeline'
+    runs-on: ubuntu-latest
+    needs: [Code-Formatter, Test-Application]
```

Comment:
This only needs `Test-Application` as `Test-Application` relies on `Code-Formatter`

Reply:
yeah that makes sense, I will change it right away